cluster/manager.go

```diff
@@ -6,9 +6,35 @@ import (
 	"time"
 
 	"github.com/hashicorp/memberlist"
+	"github.com/raintank/metrictank/stats"
 	"github.com/raintank/worldping-api/pkg/log"
 )
 
+var (
+	// total number of nodes this instance thinks are in the cluster.
```

Comment:
if you use the proper format then you can use metrics2docs to generate docs/metrics.md
see https://github.com/Dieterbe/metrics2docs#doc-format
(note the tool may not work perfectly and sometimes tries to make other changes to the file as well. but you can just use `git checkout -p docs/metrics.md` to undo that, or `git add -p docs/metrics.md` to only commit the appropriate changes)